### ISSUE TITLE
TextureProjection op does not work after patch save

### ISSUE THREAD

**Issue by esperanc** [cables-gl/cables_docs#880]
**Link to simple, reproducible example patch**
https://cables.gl/edit/CvnVip

**Describe the bug**
In the example patch, a striped pattern is projected orthogonally to an animated surface. Just after creating the TextureProjection op and placing it in the pipeline, the terrain is shown as below.

![Screenshot 2024-10-22 at 15 23 37](https://github.com/user-attachments/assets/83587107-49e6-49c5-8de5-6b418d8c6930)

However, after saving it and reloading the patch, the striped pattern disappears:

![Screenshot 2024-10-22 at 15 24 07](https://github.com/user-attachments/assets/d821bf5a-1cc1-4fd9-b926-eb053b0b7dd2)

 (This is what you will probably see if you load the example patch).

**How To Reproduce**

1. Perform the procedure above.
2. To restore the correct behavior,  copying and paste the TextureProjection op, delete the old op and rewire the copied op in the same place where the old op was.

**Platform**
 - Chrome on Mac

**Dev Tools**
Nothing useful there.

**Comment by pandrr** [comment]
yes there is a bug, i fixed it experimently on dev.cables.gl, hopefully it will not break other things...

but your patch is not correct: the projection must be below the displace.
because all height the axis values are all 0 before and so the projection does not work, move it below the displacement and it works on dev.cables.gl

**Comment by steam0r** [comment]
this made it to cables.gl with the latest release, just now!
